CHANGES.rst

```diff
@@ -1,11 +1,38 @@
 CHANGES
 =======
 
-1.2.1 (unreleased)
+1.3.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Added Django 1.7 compatibility (Antoine Catton, Rocky Meza, #35)
+- **Backwards Incompatible:** Can't pass the ``next`` URL as a POST parameter
```

Comment:
This seems like breaking stuff for no reason. Is it that hard to write `request.POST.get('next', request.GET.get('next'))`?